server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -123,7 +123,10 @@ export class AlfredResources implements core.IResources {
 }
 
 export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredResources> {
-	public async create(config: Provider): Promise<AlfredResources> {
+	public async create(
+		config: Provider,
+		customization?: Record<string, any>,
```

Comment:
Do we plan to support other scenarios than customizing the `documentRepository`? Is that why we wrap all that into this customization object? If yes, would `customizations` be a better name?

Reply:
Addressed